bftengine/tests/simpleKVBCTests/TesterReplica/main.cpp

```diff
@@ -153,8 +157,10 @@ int main(int argc, char **argv) {
 	c.replicaId = rp.replicaId;
 	c.fVal = replicaConfig.fVal;
 	c.cVal = replicaConfig.cVal;
-	c.numOfClientProxies = rp.numOfClients;
-	c.statusReportTimerMillisec = 20 * 1000;
+	c.numOfClientProxies = replicaConfig.numOfClientProxies;
+        // Allow triggering of things like state transfer to occur faster in
+        // tests.
+	c.statusReportTimerMillisec = 1000;

```

Comment:
Sometimes we want to check that the replicas can continue working without sending frequent Status messages (it is mainly important when we use many replicas). I suggest using a small value only for tests that really need it.

Reply:
I changed the code to take a CLI parameter for this and added it as another commit.